src/styles/blocks/categories.scss

```diff
@@ -0,0 +1,71 @@
+.categories {
+  &__wrapper {
+    display: grid;
+    row-gap: 40px;
+    margin-bottom: 40px;
+
+    @include onTablet {
+      row-gap: 64px;
+    }
+  }
+}
+
+.category {
+  &__photos {
+    @include pageGrid;
+    margin-bottom: 16px;
+  }
+
+  &__button {
+
+    &--wide{
```

Comment:
Add a space before the opening curly brace for consistency with other class selectors.